kafka.md

```diff
@@ -41,16 +41,7 @@ If you don't specify a broker id in your docker-compose file, it will automatica
 
 ### Automatically create topics
 
-If you want to have kafka-docker automatically create topics in Kafka during
-creation, a ```KAFKA_CREATE_TOPICS``` environment variable can be
-added in ```docker-compose.yml```.
-
-Here is an example snippet from ```docker-compose.yml```:
-
-        environment:
-          KAFKA_CREATE_TOPICS: "Topic1:1:3,Topic2:1:1"
-
-```Topic 1``` will have 1 partition and 3 replicas, ```Topic 2``` will have 1 partition and 1 replica.
+If you want to have Kafka automatically create topics during creation, modify the `scripts\kafka\kafka_init.sh` script to include the topics you want to create. The script is run by a kafka init container upon startup. The default script creates a single topic `test` with 1 partition and 1 replica.
```

Comment:
Doesn't the kafka-init.sh script create all topics by default instead of just the single 'test' topic?

Reply:
Thanks for catching that! This has been corrected.